sklearn/manifold/_mds.py

```diff
@@ -58,13 +59,20 @@ def _smacof_single(
 
     eps : float, default=1e-3
         Relative tolerance with respect to stress at which to declare
-        convergence.
+        convergence. The value of `eps` should be tuned separately depending
+        on whether or not `normalized_stress` is being used.
 
     random_state : int, RandomState instance or None, default=None
         Determines the random number generator used to initialize the centers.
         Pass an int for reproducible results across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    normalized_stress : bool, default=False
+        Whether use and return normed stress value (Stress-1) instead of raw
+        stress calculated by default. Only supported in non-metric MDS.
```

Comment:
I do not think the statement "Only supported in non-metric MDS." is not true here. The stress is normalized even if `metric=True`.

Since `smacof` itself does the validation, I am okay with not validating here. In that case, I suggest making it explicit here that the caller needs to validate:

> Only supported in non-metric MDS. The caller must ensure that if `normalized_stress=True` then `metric=False`.